release/models/system/openconfig-system-bootz.yang

```diff
@@ -0,0 +1,99 @@
+module openconfig-system-bootz {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/system-bootz";
+  prefix "oc-sys-bootz";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system { prefix oc-sys; }
+  import openconfig-types { prefix oc-types; }
+  import openconfig-yang-types { prefix oc-yang; }
+
+  organization
+    "OpenConfig working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module adds configuration and operational state relating to bootz
+    service running on a network device.";
+
+
+  oc-ext:openconfig-version "1.0.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision "2023-06-16" {
+    description
+      "Creation of bootz state paths needed for the service.";
+    reference "1.0.0";
+  }
+
+  grouping bootz-service-structural {
```

Comment:
This `grouping` needs a container in it, otherwise a duplicate `state` is being added to `/system`.

i.e.,

```yang
grouping bootz-service-structural {
   container bootz {
     ...
   }
}
```